### ISSUE TITLE
Test using `0.2` for pre-releases

### ISSUE THREAD

**Issue by fu5ha** [termhn/colstodian#4]
Would like to do a `0.2` pre-release as it's now in a "minimal viable" state. There's not a huge number of implemented encodings yet, but the most core important ones are there. Need to do a bit of real world use to iron out obvious kinks or oversights first. 

@virtualritz since you're one of the people using the `0.1` series of `colstodian`, if you'd like to take a look at the `0.2` rework (latest `main` branch, see docs link built from main in the README) and let me know your thoughts and/or take it for a test drive, that would be awesome.

**Comment by virtualritz** [comment]
I will try to have a look this weekend.

**Comment by virtualritz** [comment]
Just tagging a [Zulip discussion](https://xi.zulipchat.com/#narrow/channel/466849-color/topic/.60colstodian.60.20crate/near/482419319) about `colstodian` in regards to the new [`color`](https://github.com/linebender/color) crate here.

**Comment by virtualritz** [comment]
TLDR; the project I was using `colstodian` on was canceled.
But I recently started a new job and for that I will need something along the lines of this crate soon (again).

**Comment by virtualritz** [comment]
If you have some time, have a look at [this commit in my clone](https://github.com/fu5ha/colstodian/commit/4dddec415eb1482ccfb70fc36e19ce3e24534514). 

It's mostly stuff done by Claude, I only reviewed superficially. TLDR; I'm gonna use `colstodian` for some image processing crate I'm working on, so I need it to be in certain shape.

If you think this eventually warrants a PR from me/new release, let me know. And: no rush. :grin:
